### PR TITLE
PYD-29 Add set attachment field to none.

### PR DESCRIPTION
**Pull request by Frontman113**

### CODE REVIEW

**Comment by nick-allen** [comment]
Docs

File: swimlane/core/fields/attachment.py [hunk 1]
```diff
@@ -45,3 +46,11 @@ def get_initial_elements(self):
         raw_value = self.get_swimlane() or []
 
         return [Attachment(self.record._swimlane, raw) for raw in raw_value]
+
+    def set_python(self, value):
+        """Override to remove key from raw data when empty to work with server 2.16+ validation"""
+        if value is None:
+            super(CursorField, self).set_python([])
+            self._cursor = None
```
Comment (nick-allen) [inline swimlane/core/fields/attachment.py]: Move `self._cursor = None` to `_set` method

File: tests/fields/test_attachment.py [hunk 2]
```diff
@@ -41,3 +42,15 @@ def test_attachment_field(mock_swimlane, mock_record):
...
+
+
+def test_attachment_remove(mock_record):
+    attachments = mock_record['PCAP Attachment']
+
+    with pytest.raises(ValidationError):
+        mock_record['PCAP Attachment'] = 'invalid'
+
+    mock_record['PCAP Attachment'] = None
+    assert mock_record._raw['values']['aep2e'] == []
```
Comment (nick-allen) [inline tests/fields/test_attachment.py]: Add assert len 1 and 0 for before and after.

**Comment by Frontman113** [comment]
Pull request moved to https://github.com/swimlane/swimlane-python/pull/19. Branch on swimlane repo instead of fork for easier maintenance

**Pull request closed without merging**